### DISCUSSION TITLE
Intel ARC Graphics Card

### DISCUSSION THREAD

**Discussion by devanshdp04** [lllyasviel/Fooocus#3214] [Q&A] [answered]
I have a 8GB integrated intel ARC Graphics in my Intel Core ULTRA 5 125H processor. So i do not use the NVIDIA graphics card. And that's why it was showing that no NVIDIA graphics/driver found on your  system.So i wanted to ask that does this work only on NVIDIA/AMD graphics card if yes then is there anyway i can work around this problem.I would be very grateful you can help me out  so that I can use the FOOCUS on my local machine. 

![image](https://github.com/lllyasviel/Fooocus/assets/172265746/262f3bc5-fb1b-4e6f-82a2-616451e9731c)
![image](https://github.com/lllyasviel/Fooocus/assets/172265746/f6101268-b336-4e5a-872c-59ed66a3fc6e)

**Comment by mashb1t** [comment, marked as answer]
=> https://github.com/lllyasviel/Fooocus/pull/2120

The issue is that ARC support is not official, so handle with upmost care, and your RAM and VRAM are most likely the same thing and you'll be bottlenecked by this.

**Reply by mashb1t** [reply]
You can always use `--always-cpu` though, but also then you're 16GB away from the minimum system requirements of 32GB RAM.
Also see https://github.com/lllyasviel/Fooocus/discussions/3204#discussioncomment-9918595
